docs/pytorch.md

```diff
@@ -0,0 +1,107 @@
+# PyTorch's model with Xplique
+
+- [**PyTorch's model**: Getting started](https://colab.research.google.com/drive/1bMlO29_0K3YnTQBbbyKQyRfo8YjvDbhe)<sub> [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1bMlO29_0K3YnTQBbbyKQyRfo8YjvDbhe) </sub>
+
+## Is it possible to use Xplique with PyTorch's model ?
+
+**Yes**, it is! Even though the library was mainly designed to be a Tensorflow toolbox we have been working on a very practical wrapper to facilitate the integration of your PyTorch's model into Xplique's framework!
+
+### Quickstart
+```python
+import torch
+
+from xplique.wrappers import TorchWrapper
+from xplique.attributions import Saliency
+from xplique.metrics import Deletion
+
+# load images, labels and model
+# ...
+
+device = 'cuda' if torch.cuda.is_available() else 'cpu'
+wrapped_model = TorchWrapper(torch_model, device)
+
+explainer = Saliency(wrapped_model)
+explanations = explainer(inputs, labels)
+
+metric = Deletion(wrapped_model, inputs, labels)
+score_saliency = metric(explanations)
+```
+
+## Does it work for every modules ?
+
+It has been tested on both the `attributions` and the `metrics` modules.
+
+## Does it work for all attribution methods ?
+
+Not yet, but it works for most of them:
+
+| **Attribution Method** | PyTorch compatible |
+| :--------------------- | :----------------: |
+| Deconvolution          | ❌                |
+| Grad-CAM               | ❌                |
+| Grad-CAM++             | ❌                |
+| Gradient Input         | ✅                |
+| Guided Backprop        | ❌                |
+| Integrated Gradients   | ✅                |
+| Kernel SHAP            | ✅                |
+| Lime                   | ✅                |
+| Occlusion              | ✅                |
+| Rise                   | ✅                |
+| Saliency               | ✅                |
+| SmoothGrad             | ✅                |
+| SquareGrad             | ✅                |
+| VarGrad                | ✅                |
+| Sobol Attribution      | ✅                |
+| Hsic Attribution       | ✅                |
+
+## How does it work ?
+
+### 1. Pre-process the inputs
```

Comment:
I am not sure to understand what you want since I do not talk at al about how the gradient is computed.. I will tag this conversation as resolved but if you believe it is a significant issue feel free to unresolve it